google/ggrc-core
# GGRC-1953 Adjust Discard confirmation modal

# Issue description

"Discard Changes" popup: rename "Cancel" button to "Continue Editing" and replace "Discard" button

# Steps to test the changes

Steps to reproduce:
1. Invoke Control modal window
2. Type title and click [x] button
2. "Discard Changes" popup appears
Expected Result: Improve buttons according to mockup

# Solution description

1) Changed `confirm` method to use `confirm_cancel_buttons.mustache` by default (currently we don't use `confirm` with default params, but I fixed it just in case)
2) Renamed and edited existing `modals/confirm_buttons.mustache` that was used **ONLY** for "Discard changes" modal.
3) Changed wording, because `BUTTON_VIEW_CONFIRM_CANCEL` contained `confirm_buttons` but `confirm_cancel_buttons` template was in `BUTTON_VIEW_OK_CLOSE`.

# Sanity checklist




# PR Review checklist
